docs/api/navigation-history.md

```diff
@@ -65,3 +62,24 @@ Navigates to the specified offset from the current entry.
 #### `navigationHistory.length()`
 
 Returns `Integer` - History length.
+
+#### `navigationHistory.deleteEntryAtIndex(index)`
```

Comment:
Similar to the comment about `getAll()`, we typically prefer "remove" over "delete" when an object itself is not being destructed.

Here's a few examples:
- `session.removeExtension(extension_id)`
- `request.removeHeader(name)`
- `commandLine.removeSwitch(switch)`
- `crashReporter.removeExtraParameter(key)`
- `ipcMain.removeHandler(channel)`

Reply:
I renamed it to `removeEntryAtIndex()` to be consistent.